client/me/account-close/test/main.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * @jest-environment jsdom
+ */
+import { render, screen } from '@testing-library/react';
+import { Provider as ReduxProvider } from 'react-redux';
+import { createReduxStore } from 'calypso/state';
+import AccountSettingsClose from '../main';
+
+describe( 'AccountSettingsClose', () => {
+	it( 'Shows Manage purchases button when refundable purchases exist', () => {
+		render(
+			<ReduxProvider store={ createTestStore( true, true ) }>
+				<AccountSettingsClose />
+			</ReduxProvider>
+		);
+		expect( screen.queryByTestId( 'manage-purchases-button' ) ).toBeInTheDocument();
+	} );
+
+	it( 'Tells user to wait if Atomic site is being deleted', () => {
```

Comment:
Nit: It may be more accurate to describe this case as "Tells user to wait if they still have an Atomic site" -- despite us showing the message because we're waiting, the actual driving condition is that the user still has an Atomic site.

This is not blocking, and pretty minor.

Reply:
Ok, done